README.md

```diff
@@ -26,4 +26,4 @@ The application is built using Kotlin.
 
 ## License
 
-This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details.
+This project is licensed under the MIT License - ~~see the LICENSE.md file for details.~~
```

Comment:
I think we should add the Apache 2.0 license to this project. I's almost the same as MIT, but with legal conditions to our logo's etc. See choosealicense.com

Reply:
I'm using it for my written guides.